src/components/AppHeader.tsx

```diff
@@ -134,4 +142,9 @@ const HeaderContainer = styled(motion.header)`
   }
 `
 
+const OfflineIcon = styled(WifiOff)`
+  color: ${({ theme }) => theme.font.secondary};
+  margin: 0 10px !important;
```

Comment:
Everything seems to work fine without the `!important` as well, so let's avoid it when it's not needed.